mitodl/mit-open
# Use new OLL column for secondary topic; fix semester capitalization mismatch

### What are the relevant tickets?
- Closes https://github.com/mitodl/hq/issues/5379
- Closes https://github.com/mitodl/hq/issues/5380

### Description (What does it do?)
- Update the OLL ETL pipeline to use a new column in the google sheet for secondary topics (and updates the downloaded csv included in the repo)
- Update `learning_resources.utils.get_year_and_semester` and `learning_resources.constants.semester_mapping` so that semester values are consistently capitalized.


### How can this be tested?
Run the following, it should complete without errors:
```
./manage.py backpopulate_oll_data
```

You can also try this to ingest directly from the google sheet instead of the csv provided in this repo:
```
./manage.py backpopulate_oll_data --sheets_id 1bWS-mqmuVraseMfY2YRu6gRnhXnjAS2N8CLiw4Tl8PE
```

Afterwards make sure that 60 results are returned at http://open.odl.local:8062/search/?platform=oll and that the OCW courses ingested from OLL all have capitalized first letters in their semester values.
